### ISSUE TITLE
Talkback will not read all numbers in phone mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#124]
```
What steps will reproduce the problem?
1. installed talkback v 3.1.3
2. rebooted the phone with the power button
3. bring up the phone feature to dial, it will sound proper for the fist few 
numbers and then after that, all you you hear is click click click.  If you 
earease the entire number you can hear the whole number.

What is the expected output? What do you see instead?
N/A

What version of the product are you using? On what operating system?
talkback 3.1.1 
phone LG-P500
android 2.2.2
kernal 2.6..32.9

Please provide any additional information below.

The previous version worked without any problems,  this happened after the 
manual update.  This is used for a visually impaired client, whom can not use 
her phone now.  She has very little sight.  Without this program, she doesn't 
even want to use her phone.
```

Original issue reported on code.google.com by `landrees...@gmail.com` on 20 Aug 2012 at 7:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 13 Sep 2012 at 6:35

**Comment by GoogleCodeExporter** [comment]
```
TalkBack is speaking the phone's text formatting events, which are flushing the 
typed number.
```

Original comment by `alanv@google.com` on 13 Sep 2012 at 6:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
It only speaks the first few numbers only. Is there anything I need to do to 
help troubleshoot this issue??  This is a very important program.  It will not 
even read back the text messages.  The previous version worked.  It will not 
allow me to reinstall an earlier verison.
```

Original comment by `landrees...@gmail.com` on 14 Sep 2012 at 11:08

**Comment by GoogleCodeExporter** [comment]
```
We have identified a fix and will include it in the next public beta.
```

Original comment by `alanv@google.com` on 14 Sep 2012 at 11:14

**Comment by GoogleCodeExporter** [comment]
```
Do you know when will that be?  Customer is wanting to use her phone ASAP.
thanks
```

Original comment by `landrees...@gmail.com` on 14 Sep 2012 at 11:20

**Comment by GoogleCodeExporter** [comment]
```
Resolved in TalkBack v3.2.1 available on Play Store.
```

Original comment by `alanv@google.com` on 7 Dec 2012 at 6:25
- Changed state: **Fixed**
